rhoas-cli-kafka/README.adoc

```diff
@@ -208,20 +208,19 @@ After creating a Kafka instance, you can create Kafka topics to start producing
 
 * You have created a Kafka instance, and it has a `ready` status.
 
-.Procedure
-
-. Verify that you are using the Kafka instance that you created.
-+
---
-This example switches to the `my-kafka` instance that was previously created.
+[NOTE]
+====
+You can use `rhoas` to switch to a specific Kafka instance.
```

Comment:
`rhoas` => `rhoas service-registry use`

Reply:
We mean the `rhoas` tool here. But your suggestion works. Updated